### ISSUE TITLE
Recommendation: Right-Click Copy Image

### ISSUE THREAD

**Issue by astrofinesse** [keybase/client#24576]
I love keybase and think it has a bright future but not being able to right click an image and copy to my clipboard is a big deal breaking right now.

**Comment by kiwimato** [comment]
YES! That would be awesome. As a workaround I use Flameshot on Linux or Greenshot on Windows to simply take a screenshot. 
Sadly it seems Keybase doesn't have support anymore and rarely there are any updates :(
